components/nwhacks-footer/nwhacks-footer.html

```diff
@@ -1,48 +1,71 @@
 <dom-module id="nwhacks-footer">
-  <template>
-    <footer >
-      <div class="skyline">
-        <object data="/svg/Skyline-Footer-Cropped.svg" type="image/svg+xml"></object>
-      </div>
-      <table class="foot">
-        <tbody>
-          <tr>
-            <td class="small-middle">
-              <nav class="grid-container">
-                nwHacks Team
+    <template>
 
-                <a href="https://github.com/nwhacks">Source Code</a>
-              </nav>
-              <br>
-            </td>
-            <td class="small-first">
-              <nav class="grid-container">
-                <a href="mailto:apply@nwhacks.io">Email Us</a>
-                <a href="http://mlh.io/code-of-conduct">Code of Conduct </a>
-                <a href="/sponsors#sponsor">Become a Sponsor</a>
-                <a href="mailto:sponsorship@nwhacks.io">Email Sponsorship</a>
-              </nav>
-              <br>
-            </td>
-            <td class="small-middle">
-              <a href="https://www.facebook.com/nwHacks">Facebook</a>
-              <a href="https://twitter.com/nwHacks">Twitter</a>
-              <a href="https://medium.com/@nwHacks">Medium</a>
-              <a href="https://www.instagram.com/nwhacks_">Instagram</a>
-              <br>
-            </td>
-            <td class="small-last">
-              Organised and held by <a href="http://www.startupstorm.org/" target="_blank">StartupStorm</a>
-            </td>
-          </tr>
-          <tbody>
-      </table>
-    </footer>
+        <footer>
+            <div class="foot">
+                <div class="social">
+                    <a href="https://www.facebook.com/nwHacks"><img src="/svg/fb_icon.svg" class="icon"/></a>
+                    <a href="https://www.instagram.com/nwhacks_"><img src="/svg/insta_icon.svg" class="icon"/></a>
+                    <a href="https://medium.com/@nwHacks"><img src="/svg/medium_icon.svg" class="icon"/></a>
+                    <a href="https://twitter.com/nwHacks"><img src="/svg/twitter_icon.svg" class="icon-last"/></a>
+                </div>
+                <div class="links">
+                    <div class="link"><a href="mailto:apply@nwhacks.io">E-mail Us</a></div>
+                    <div class="link"><a href="http://mlh.io/code-of-conduct">Code of Conduct</a></div>
+                    <div class="link-last"><a href="/sponsors#sponsor">Become a Sponsor</a></div>
+                </div>
+                <p>
+                    Organized and help by <u id="org-link"><a href="https://www.startupstorm.org/">StartupStorm</a></u>
+                </p>
+                <p style="font-weight: lighter;">
+                    Copyright © 2017 nwHacks
+                </p>
+            </div>
+        </footer>
 
-  </template>
-  <script>
-Polymer({
-  is: "nwhacks-footer",
-});
-  </script>
+        <!--<footer >-->
```

Comment:
delete old code.

No need to preserve it since you can just use git history on the off chance you need it.